### DISCUSSION TITLE
does `z.discriminatedUnion` support nested paths on the discriminator?

### DISCUSSION THREAD

**Discussion by stefanprobst** [colinhacks/zod#969] [Q&A] [answered]
hi, quick question: does `discriminatedUnion` support nested paths on the discriminator, i.e. `z.discriminatedUnion("type.type", [ ... ])` or `z.discriminatedUnion(["type", "type"], [ ... ])`? thanks

**Comment by JacobWeisenburger** [comment, marked as answer]
Currently, no. But PR's are always welcome. This could be a cool use case.
